docs/get-started/arduino.md

```diff
@@ -45,9 +45,12 @@ LVGL has its own configuration file called `lv_conf.h`. When LVGL is installed t
 
 ## Initialize LVGL and run an example
 
-Take a look at [LVGL_Arduino.ino](https://github.com/lvgl/lvgl/blob/master/examples/LVGL_Arduino.ino) to see how to initialize LVGL.
+Take a look at [LVGL_Arduino.ino](https://github.com/lvgl/lv_arduino) to see how to initialize LVGL.
+Note, that this repository is obsolete and can be used only as an example of initializing the library with a given display driver.
```

Comment:
> Note, that this repository is obsolete

What do you mean by **this** repository? Which repository?

Reply:
Here I mean https://github.com/lvgl/lv_arduino

Reply:
Why do we need to mention this repo?

Reply:
It seems, that it was previously used. I can change it to the correct repository with examples for Arduino.

Reply:
IMO the last sentence is no required. A link to the Arduino.ino should be enough to point the users in the right direction.